incubator/oauth-proxy/templates/deployment.yaml

```diff
@@ -62,6 +65,16 @@ spec:
             port: {{ .Values.service.internalPort }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
+{{- if .Values.config.configFile }}
+        volumeMounts:
+        - mountPath: /etc/oauth2_proxy
+          name: {{ template "oauth-proxy.fullname" . }}-configmap
```

Comment:
`oauth2-proxy.fullname`

Reply:
I've taken care of this in PR: https://github.com/kubernetes/charts/pull/4744